pgjdbc/src/main/java/org/postgresql/PGProperty.java

```diff
@@ -637,6 +637,15 @@ public enum PGProperty {
       "0",
       "The timeout value in seconds max(2147484) used for socket read operations."),
 
+  /**
+   * If true it uses time stamp with time zone type always for java sql Timestamp
```

Comment:
From what I can see in this PR the only time it forces timestamp with timezone is for setNull